### PR TITLE
less cleanup to reduce warnings by stylelint

### PR DESCRIPTION
**Pull request by StefanSpieker**
Another small cleanup of less syntax to satisfy stylelint. The only thing beyond was a removal of a duplicate property.

### Proposed changelog entries

* N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by fqueiruga** [comment]
What property was removed?

While I'm in favour of this PR and I think it was due to happen, I think it should be accompanied with a run with a code formatter to get some consistent whitespace use. Otherwise we will end up doing another one in the future.

**Comment by timja** [comment]
I would like to get https://github.com/jenkinsci/jenkins/pull/3895 merged before we go too crazy formatting these files (I agree it definitely needs doing though)

File: war/src/main/less/pluginSetupWizard.less
Comment (StefanSpieker) [inline war/src/main/less/pluginSetupWizard.less:495]: @fqueiruga ``outline:none`` was removed, because it was duplicated in the block.

**Comment by fqueiruga** [comment]
+1 to what Tim said, I'd suggest putting this on hold

**Comment by fqueiruga** [comment]
Right now this PR will conflict with many open efforts. I would suggest removing the changes on colors.less and style.less, as those are the problematic files. Then it could be merged IMO.

**Comment by timja** [comment]
This can be re-evaluated once https://github.com/jenkinsci/jenkins/pull/4820 is merged (should be this week)

**Comment by oleg-nenashev** [comment]
OK, now we have a merge conflict

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
merge conflict needs to be resolved

**Comment by timja** [comment]
> OK, now we have a merge conflict

expected, which is why it was on-hold, that's now been merged in so this is unblocked

**Comment by StefanSpieker** [comment]
merged with master and resolved all conflicts.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
conflicts again

**Comment by StefanSpieker** [comment]
merged again with master and resolved all conflicts.

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Thanks!  We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**
